src/pages/home/report/ReportActionCompose.js

```diff
@@ -599,7 +604,12 @@ class ReportActionCompose extends React.Component {
         const mentionCode = `@${mentionObject.alternateText}`;
         const commentAfterAtSignWithMentionRemoved = this.state.value.slice(this.state.atSignIndex).replace(CONST.REGEX.MENTION_REPLACER, '');
 
-        this.updateComment(`${commentBeforeAtSign}${mentionCode} ${commentAfterAtSignWithMentionRemoved}`, true);
+        this.updateComment(
+            `${commentBeforeAtSign}${mentionCode} ${
+                commentAfterAtSignWithMentionRemoved.slice(0, 1) === ' ' ? commentAfterAtSignWithMentionRemoved.slice(1) : commentAfterAtSignWithMentionRemoved
```

Comment:
Given that we're doing the exact same thing on both line 609 and 576, could we move this into some sort of helper function?